tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2608,7 +2645,39 @@ private String handleDateFormatItem(String xpath, String value, boolean showCont
                 DateFormatSymbols dfs = sdf.getDateFormatSymbols();
                 dfs.setTimeSeparatorString(timeSeparator);
                 sdf.setDateFormatSymbols(dfs);
-                if (id == null || id.indexOf('B') < 0) {
+                if (parts.contains("availableFormats")) {
+                    String example = addExampleResult(sdf.format(DATE_SAMPLE) + ", " + sdf.format(DATE_SAMPLE5), "", showContexts);
+                    String altId = generateAltPattern(id);
+                    if (!altId.equals(id)) {
+                        String altXpath = "//ldml/dates/calendars/calendar[@type=\""
+                            + calendar
+                            + "\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\""
+                            + altId
+                            + "\"]";
+                        String forcedAltValue = cldrFile.getWinningValue(altXpath);
+                        String unforcedAltValue = generateAltPattern(value);
+                        String forcePrefix = forcedAltValue != null ? "" : EXAMPLE_OF_INCORRECT;
```

Comment:
Originally, I had the condition as `forcedAltValue.equals(unforcedAltValue)`
That didn't work (forcedAltValue was sometimes `null`), so I tried this instead. Does this make sense?

Reply:
This condition is supposed to determine whether or not a value is forced

Reply:
If it does make sense, I realize I've flipped the condition (forcing would be when the value is not null, not when the value is null)

However, I'm very dubious of this, since it's not what we discussed. As written, it seems like forcedAltValue is always null.